File: lang_java/Aksesoris.java
public class Aksesoris extends Petshop{
// membuat class Aksesoris sebagai anak dari class Petshop

    private String jenisProduk;
    private String bahanProduk;
    private String warnaProduk;

    // konstruktor kosong
    public Aksesoris(){
        // memanggil konstruktor dari parent class
        super();
    }

    // konstruktor dengan parameter
    public Aksesoris(String idProduk, String namaProduk, int hargaProduk, int stokProduk, String jenisProduk, String bahanProduk, String warnaProduk){
        // memanggil konstruktor parent class
        super(idProduk, namaProduk, hargaProduk, stokProduk);
        this.jenisProduk = jenisProduk;
        this.bahanProduk = bahanProduk;
        this.warnaProduk = warnaProduk;
    }

    public void setJenisProduk(String jenisProduk){
        // setter untuk mengisi jenisProduk
        this.jenisProduk = jenisProduk;
    }

    public String getJenisProduk(){
        // getter untuk mendapatkan jenisProduk
        return jenisProduk;
    }

    public void setBahanProduk(String bahanProduk){
        // setter untuk mengisi bahanProduk
        this.bahanProduk = bahanProduk;
    }

    public String getBahanProduk(){
        // getter untuk mendapatkan bahanProduk
        return bahanProduk;
    }

    public void setWarnaProduk(String warnaProduk){
        // setter untuk mengisi warnaProduk
        this.warnaProduk = warnaProduk;
    }

    public String getWarnaProduk(){
        // getter untuk mendapatkan warnaProduk
        return warnaProduk;
    }

}

File: lang_py/Aksesoris.py
from Petshop import Petshop

class Aksesoris(Petshop):
# membuat class Aksesoris sebagai anak dari class Petshop

    def __init__(self, idProduk, namaProduk, hargaProduk, stokProduk, jenisAksesoris, bahanAksesoris, warnaAksesoris):
        # konstruktor
        super().__init__(idProduk, namaProduk, hargaProduk, stokProduk)
        self.jenisAksesoris = jenisAksesoris
        self.bahanAksesoris = bahanAksesoris
        self.warnaAksesoris = warnaAksesoris

    def setJenisAksesoris(self, jenisAksesoris):
        # mengeset jenisAksesoris
        self.jenisAksesoris = jenisAksesoris

    def getJenisAksesoris(self):
        # mengembalikan jenisAksesoris
        return self.jenisAksesoris

    def setBahanAksesoris(self, bahanAksesoris):
        # mengeset bahanAksesoris
        self.bahanAksesoris = bahanAksesoris

    def getBahanAksesoris(self):
        # mengembalikan bahanAksesoris
        return self.bahanAksesoris
    
    def setWarnaAksesoris(self, warnaAksesoris):
        # mengeset warnaAksesoris
        self.warnaAksesoris = warnaAksesoris

    def getWarnaAksesoris(self):
        # mengembalikan warnaAksesoris
        return self.warnaAksesoris